### ISSUE TITLE
Colors aren't parsed correctly in topic.

### ISSUE THREAD

**Issue by blasphemy** [hexchat/hexchat#150]
If you turn off color stripping for topics, it doesn't parse the colors, only boxes appear.
![example](http://i.imgur.com/CY9uz.png)

**Comment by GTAXL** [comment]
Yes, I have the same issue too and it irritates me as colors in the topic seems like a cool feature. Can you please fix this and provide an update on the status? Here is what I get, http://pik.gtaxl.net/24_11_15_20_12_21.png using charset UTF-8 and font Arial Bold 12. As you can see the color is fine in the chat section with the same font and charset, it's just in the topic bar it displays the color codes used to create said color, which are supposed to be hidden.

**Comment by samcv** [comment]
![screenshot_20161012_004158](https://cloud.githubusercontent.com/assets/20145557/19301611/279bb176-9015-11e6-9941-12f562e7a3f9.png)
This is still a issue.  Is there any update on the status of this?  This is my main annoyance with hexchat.

**Comment by b0nk** [comment]
Still an issue in 2.14.2
